### PR TITLE
[Readme]Reduce space when display content in  readme container 

### PR DESCRIPTION
**Pull request by lyndaidaii**
Summary of the changes (in less than 80 characters):

Reduce space when display readme 

Before: 
uploadbefore
manageReadme

After: 
uploadPageAfter
editRee

Addresses https://github.com/NuGet/NuGetGallery/issues/8508, https://github.com/NuGet/NuGetGallery/issues/8651

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/_ImportReadMe.cshtml [hunk 1]
```diff
@@ -87,7 +87,7 @@
             </div>
         </div>
 
-        <div class="hidden" id="readme-preview" style="padding-top:3em;">
+        <div class="hidden" id="readme-preview" style="padding-top:0.25em;">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/_ImportReadMe.cshtml:90]: Could we move this inline CSS to a LESS file that is appropriate for all of the places `_ImportReMe.cshtml` is used?

**Comment by lyndaidaii** [comment]
> It seems that there is also a lot of extra space below the content. Is that intentional?
> image

This is intential, the readme container size has default size.

**Comment by loic-sharma** [comment]
> This is intential, the readme container size has default size.

What's the purpose of the container's default size? The forced size looks a little strange to me.

**Comment by lyndaidaii** [comment]
> > This is intential, the readme container size has default size.
> 
> What's the purpose of the container's default size? The forced size looks a little strange to me.

@chgill-MSFT, could you help provide some idea in terms of extra space below content?

**Comment by lyndaidaii** [comment]
image

The space here actually is div for edit markdown
https://github.com/NuGet/NuGetGallery/blob/main/src/NuGetGallery/Views/Packages/_ImportReadMe.cshtml#L96
When we preview readme content, we remove hidden class
https://github.com/NuGet/NuGetGallery/blob/main/src/NuGetGallery/Scripts/gallery/page-edit-readme.js#L332

I created new ticket to address seperately, It was not trivial fix, here is ticket #https://github.com/NuGet/NuGetGallery/issues/8651

**Comment by lyndaidaii** [comment]
After syncing with Joel, we found alternative approach to clean up space issue below content.  This PR will address both #8508, #8651

**Review by joelverhagen — APPROVED** [review]

File: src/Bootstrap/less/theme/common-readme.less [hunk 2]
```diff
@@ -32,7 +32,19 @@
...
 }
+
+#edit-markdown {
+  padding-top: 3em;
+  padding-bottom: 3em;
+}
+
+#preview-html {
+  padding-top: 3em;
+  padding-bottom: 3em
```
Comment (loic-sharma) [inline src/Bootstrap/less/theme/common-readme.less:49]:
```suggestion
  padding-bottom: 3em;
```

**Review by loic-sharma — APPROVED** [review]

**Pull request merged**